src/Servers/Kestrel/test/Interop.FunctionalTests/H2SpecTests.cs

```diff
@@ -61,6 +61,7 @@ public static TheoryData<H2SpecTestCase> H2SpecTestCases
         get
         {
             var dataset = new TheoryData<H2SpecTestCase>();
+            var toSkip = new string[] { "http2/6.9.2/2" };
```

Comment:
Seems like this ought to be a map from name to reason?  Alternatively, it could be a simple string.